ONSdigital/dis-wagtail
# Main Menu

### What is the context of this PR?

Add a navigation app in Wagtail to manage the main menu, allowing for structured, customisable navigation.

### How to review

- Spin up the ONS BETA Web App Locally as normal.
- Snippets -> Main Menu -> Add main menu.
- Fill in the menu content -> After saving every time you should see the preview on the preview panel.
- Go to Settings -> Navigation Settings ->  Choose main menu -> Choose the menu shown so it can be displayed on the website.

Loom walkthrough of the main menu: [Main Menu Features Loom Walkthrough](https://www.loom.com/share/8d9f8f883a194448ab125332e80547f3?sid=01b18433-87d8-40f9-8796-b8c8d973ddd4)

Credit: @zerolab for the help and guidance.

### Follow-up Actions

Fix column alignment issues (can be seen during the loom video): https://github.com/ONSdigital/dis-wagtail/pull/63
